lib/rules/symbol-description.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @fileoverview Rule to enforce description with the `Symbol` object
+ * @author Jarek Rencz
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const astUtils = require("../ast-utils");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "require symbol descriptions",
+            category: "ECMAScript 6",
+            recommended: false
+        },
+
+        schema: []
+    },
+
+    create(context) {
+
+        /**
+         * Reports if node does not conform the rule in case rule is set to
+         * report missing description
+         *
+         * @param {ASTNode} node - A CallExpression node to check.
+         * @returns {void}
+         */
+        function checkArgument(node) {
+            if (node.arguments.length === 0) {
+                context.report({
+                    node,
+                    message: "Expected Symbol to have a description."
+                });
+            }
+        }
+
+        return {
+            "Program:exit": () => {
```

Comment:
Should `object-shorthand` (which we have on in the codebase I think?) catch this? I guess it might not due to `this` working differently...